### PR TITLE
feat(ui-compo): Migrate Sidebar Component

### PR DESCRIPTION
**Pull request by kengork**
# PR Prelude

Thank you for contributing to sheikah! :)

**Please fill in this template (by putting an `x` inside
the brackets) _before_ filing your PR:**

- [x] I have read and understood [CODE_OF_CONDUCT][code] document.
- [x] I have read and understood [CONTRIBUTING][cont] document.
- [x] I have read and understood [STYLEGUIDE][style] document.
- [x] I have included tests for the changes in my PR. If not, I have included a
  rationale for why I haven't.
- [x] My code is formatted and is accepted by `yarn fmt`.
- [x] **I understand my PR may be closed if it becomes obvious I didn't
  actually perform all of these steps.**

# Why this change is necessary and useful

[Please explain **in detail** why the changes in this PR are needed.]

[code]: https://github.com/witnet/sheikah/blob/master/.github/CODE_OF_CONDUCT.md
[cont]: https://github.com/witnet/sheikah/blob/master/.github/CONTRIBUTING.md
[style]: https://github.com/witnet/sheikah/blob/master/docs/STYLEGUIDE.md

### CODE REVIEW

File: src/styles/app.global.scss [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+
+  body {
+    bottom: 0;
+    font-family: $font-family;
+    left: 0;
+    position: fixed;
+    right: 0;
+    top: 0;
+
+    .ant-select-selection-selected-value {
```
Comment (Tommytrg) [inline src/styles/app.global.scss]: We don't need _antd_ classes anymore

File: src/views/Home.vue [hunk 2]
```diff
@@ -92,62 +28,18 @@ export default {
...
-
-.grid {
-  box-shadow: 0 1px 5px 1px rgba(0, 0, 0, 0.1);
-  display: grid;
-  grid-template-columns: 30% 1fr;
-  height: 100%;
-}
-
-.sidebar {
+@import url('https://fonts.googleapis.com/css?family=Titillium+Web:300,300i,400,400i,600,600i&display=swap');
```
Comment (Tommytrg) [inline src/views/Home.vue]: Add blank line

File: src/views/Home.vue [hunk 3]
```diff
@@ -1,83 +1,19 @@
...
-        </el-menu-item>
-      </el-menu>
-    </el-aside>
-
-    <router-view></router-view>
-  </el-container> -->
 </template>
 
 <script>
+import Sidebar from '../components/Sidebar.vue'
```
Comment (Tommytrg) [inline src/views/Home.vue]:
```suggestion
import Sidebar from '@/components/Sidebar.vue'
```

File: src/components/Sidebar.vue [hunk 4]
```diff
@@ -69,3 +55,142 @@ export default {
...
+      flex-flow: row nowrap;
+      font-size: $sidebar-link-font_size;
+      font-weight: $sidebar-link-font_weight;
+      height: 40px;
+      margin: 0px 15px 0px 0px;
+      padding: 8px;
+      text-decoration: none;
+      width: 100%;
+      text-align: left;
+      .router-link-active{
```
Comment (aesedepece) [inline src/components/Sidebar.vue]:
```suggestion
      &.router-link-active{
```

**Comment by aesedepece** [comment]
- `.link` has redundant `margin-right`
- we need to reset everything to `box-sizing: border-box`. Otherwise, using `width: 100%` is a true mess.

**Pull request merged**